packages/validator/test/cli-validator/mock-files/oas3/test-formats.yaml

```diff
@@ -0,0 +1,69 @@
+openapi: 3.0.0
+servers:
+  - url: https://typeformats.com/api
+components:
+    post:
```

Comment:
I know this is just a test case but it surprised me to see "post" underneath "components" - I wonder if it should be a more "street-legal" API def for the test, but it's not a big deal either way 🙂

Reply:
I wish I could say "That was just a test to make sure the reviewers were paying attention", but all I can honestly say is "Thank goodness someone was paying attention!" :)
Fixed it.

Reply:
I think it's interesting that this problem didn't prevent the validator from processing those schema properties...

Reply:
The code for this walks through the whole spec and only checks for local context when it determines it's inside of a "schema". It's unaware that the path all the way up isn't quite right.